docs/examples/tutorials/solver_onedim/02_onedim_sim_hetero_atm.py

```diff
@@ -0,0 +1,140 @@
+"""
+1D-simulations on heterogeneous atmospheres
+===========================================
+"""
+
+# %%
+# This tutorial illustrates how to perform simulations on heterogeneous
+# atmospheres using :class:`.OneDimSolverApp`.
+#
+# Setup
+# -----
+#
+# Let us work in monochromatic mode at the wavelength of 579 nm.
+# In order to focus on the atmosphere properties, we do not specify the
+# ``illumination`` and ``surface`` fields of the application configuration.
+# In other words, we use the default surface (lambertian surface with
+# 50%-reflectance) and illumination (directional illumination with 0° - zenith
+# and azimuth angles).
+#
+# For the heterogeneous atmosphere, we use the ``us76_approx`` radiative
+# properties profile, with default parameters (see the
+# :ref:`sec-atmosphere-heterogeneous` user guide page for more info).
+#
+# We will compute the top-of-atmosphere bi-directional reflectance factor in
+# the principal plane so we use the ``distant`` measure and use a
+# one-dimensional film resolution (corresponding dimension maps the zenith
+# angle coordinate).
+# We set the ``spp`` parameter value so that the results display reduced noise:
+
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+
+from eradiate.solvers.onedim import OneDimSolverApp
+
+config = {
+    "mode": {
+        "type": "mono_double",
+        "wavelength": 579.
+    },
+    "atmosphere": {
+        "type": "heterogeneous",
+        "profile": {
+            "type": "us76_approx"
+        }
+    },
+    "measures": [{
+        "type": "distant",
+        "id": "toa_plane",
+        "film_resolution": [90,1],
+        "spp": 65536
+    }]
+}
+app = OneDimSolverApp.from_dict(config)
+
+# %%
+# Run
+# ---
+# Let us run the application:
+
+app.run()
+
+# %%
+# For future uses, we define a function that plots the results:
+
+
+def visualise(results):
+    plt.xticks(np.arange(-90, 91, step=15))
+    results.brf.plot(x="vza", marker="o", markersize=2, linewidth=0.5)
+
+
+# %%
+# Let us visualise our results:
+
+results_579 = app.results["toa_plane"]
+visualise(results_579)
+
+# %%
+# We observe that the top-of-atmosphere bi-directional reflectance value
+# cluster around 0.5, which is consistent with the value of the surface's
+# reflectance.
+# Indeed, the atmosphere we have created is almost purely scattering:
+
+print(app.scene.atmosphere.profile.albedo)
```

Comment:
```suggestion
display(app.scene.atmosphere.profile.albedo)
```
should show the HTML repr, if I'm not mistaken.

Reply:
Yes, but then what if the user download the tutorial as python source code, and not as jupyter notebook?

Reply:
Fixed :)